examples/bzlmod/entry_point/BUILD.bazel

```diff
@@ -1,18 +1,38 @@
-load("@pip_39//:requirements.bzl", "entry_point")
+load("@python_versions//3.9:defs.bzl", py_entry_point_binary_3_9 = "py_entry_point_binary")
 load("@rules_python//python:defs.bzl", "py_test")
+load("@rules_python//python:py_entry_point_binary.bzl", "py_entry_point_binary")
 
-alias(
+# We can specifically request the py_entry_point_binary to be running with Python 3.9.
+py_entry_point_binary_3_9(
     name = "yamllint",
-    actual = entry_point("yamllint"),
+    pkg = "@pip//yamllint:pkg",
+    # yamllint does not have any other scripts except 'yamllint' so the
+    # user does not have to specify which console script we should chose from
+    # the package.
+)
+
+# Here we are running the default version.
```

Comment:
Please explain what a "default" version is.  Approach the documentation from a user that is new to bazel.